src/session/dummyAppSessionStateClient.ts

```diff
@@ -13,7 +15,7 @@ export function createDummyAppSessionStateClient(): AppSessionStateClient {
         currentSessionState = {
           status: "authenticated",
           dopplerAccountName: testUserData.user.email,
-          userData: testUserData as any,
+          userData: parseUserData(testUserData) as UserData,
```

Comment:
The cast should not be necessary because `parseUserData` should already return an `UserData` object.

```suggestion
          userData: parseUserData(testUserData),
```